README.MD

```diff
@@ -202,13 +202,21 @@ func ParseMessage(line string) (*User, interface{})
 These are the available methods of the client so you can get your bot going:
 
 ```go
-func (c *Client) Say(channel, text string)
-func (c *Client) Whisper(username, text string)
-func (c *Client) Join(channel string)
-func (c *Client) Depart(channel string)
-func (c *Client) Userlist(channel string) ([]string, error)
+func (c *Client) Ban(channel, username, reason string)
 func (c *Client) Connect() error
+func (c *Client) DeleteMessage(channel, msgID string)
+func (c *Client) Depart(channel string)
 func (c *Client) Disconnect() error
+func (c *Client) FollowersOff(channel string)
+func (c *Client) FollowersOn(channel, duration string)
+func (c *Client) Join(channel string)
+func (c *Client) Reply(channel, parentMsgId string, text string)
+func (c *Client) Say(channel, text string)
+func (c *Client) SetIRCToken(ircToken string)
+func (c *Client) SetJoinRateLimiter(rateLimiter RateLimiter)
+func (c *Client) Userlist(channel string) ([]string, error)
+func (c *Client) Userlist(channel string) ([]string, error)
+func (c *Client) Whisper(username, text string)
```

Comment:
Good catch that we're missing a bunch of function here, but preferably this is done as part of a separate PR

Revert the change here to only include the new function you've added